addons/survey/static/src/xml/survey_choice_image_zoomer_templates.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-name="survey.survey_choice_image_zoomer">
+        <div role="dialog" class="o_survey_img_zoomer_modal modal o_technical_modal fade d-flex align-items-center p-0"
+            aria-labbelledby="Image Zoom Dialog" tabindex="-1">
+            <div class="o_survey_img_zoomer_dialog modal-dialog h-100 w-100 mw-100 py-0" role="Picture Enlarged">
+                <div class="modal-content h-100 bg-transparent">
+                    <div class="o_survey_img_zoomer_body modal-body h-100 bg-transparent d-flex justify-content-center">
+                        <button type="button" data-bs-dismiss="modal" aria-label="close"
+                            class="o_survey_img_zoomer_close_btn close text-white btn-close-white btn-close position-absolute"/>
+                        <img t-attf-class="o_survey_img_zoomer_image img img-fluid d-block m-auto" t-att-src="widget.sourceImage" alt="Zoom on Image"/>
+                        <div class="o_survey_img_zoomer_btns position-absolute">
```

Comment:
Wondering if those need to be positioned in an absolute fashion as it makes the css a bit more complex, can't we place them after the image?

Reply:
Yes this is necessary, as the size of the image changes, the size of the screen as well (and could rotate on small screens), we could have the buttons out of sight, below the visible frame for instance. We do not want to have a vertical scroll bar. I've tried a few solutions but think this is the best way.